Catappult/appcoins-wallet-android
# [task/APPC-3526] Incomplete payment methods flow

**What does this PR do?**

   Adds a visual information in case the user has other payment methods available but are not done yet in the first payment flow

**Database changed?**

  No

**Where should the reviewer start?**

- OnboardingPaymentMethodsFragment.kt
- onboarding_incomplete_payment_methods_layout.xml

**How should this be manually tested?**

  - Install the wallet after going to the web view to install it
  - check the payment methods screen

**What are the relevant tickets?**

  https://aptoide.atlassian.net/browse/APPC-3526

**Code Review Checklist**
